codecs/codecs-ildcp/src/test/java/org/interledger/codecs/ildcp/AsnIldcpResponseCodecTest.java

```diff
@@ -59,28 +60,28 @@ public void setUp() {
   @Test
   public void decode() {
     codec.encode(RESPONSE);
-    assertThat(codec.getCodecAt(0).decode(), is(FOO_ADDRESS));
-    assertThat(codec.getCodecAt(1).decode(), is((short) 9));
-    assertThat(codec.getCodecAt(2).decode(), is(BTC));
+    assertThat(codec.getCodecAt(0).decode(), Is.is(FOO_ADDRESS));
```

Comment:
Per my comments above...

```suggestion
    assertThat(codec.getCodecAt(0).decode(), is(FOO_ADDRESS));
```

Reply:
I think the only remaining thing here is to remove all the `CoreMatchers` prefixes in the unit tests.

Reply:
removed the `CoreMatchers` in the latest commit.